src/utils/interfaces.ts

```diff
@@ -41,9 +41,28 @@ export interface IEEImage {
 }
 
 export interface IEEInfo {
-  name: string;
   id: string;
-  imageId: string;
-  posterUrl: string;
+  name: string;
   description: string;
+  imageData: {
+    [year: string]: {
+      default?: boolean;
```

Comment:
It would be better if the default field would not be optional. By doing that we force us to not forget to put it.

```suggestion
      default: boolean;
```